contributing/community/releases.rst

```diff
@@ -91,7 +91,21 @@ Below is the schedule for the first few versions that use this release model:
 * **Blue** represents the Stabilization phase
 * **Green** represents the Maintenance period
 
-This results in very predictable dates and maintenance periods:
+.. tip::
+
+    If you want to learn more about the timeline of any given Symfony version,
+    use the online `timeline calculator`_. You can also get all data as a JSON
+    string via a URL like `https://symfony.com/roadmap.json?version=2.x`.
```

Comment:
The last sentence seems to be a feature that should documented on the roadmap page, not in this reference docs.